R/eml.R

```diff
@@ -29,6 +29,61 @@ pid_to_eml_other_entity <- function(mn, pids) {
   sysmeta_to_eml_other_entity(sysmeta)
 }
 
+#' Create an EML dataTable object for a given PID. 
+#' This function generates an attributeList and physical and constructs a dataTable.
```

Comment:
Add a newline before this line so the help page has the right title and description. If you want to get really funky, wrap in-text references to code/classes/etc in `\code{}` directives like `\code{attributeList}` to make it even more clear to the user that you're talking about a specific { EML object | XML element | class }.

Reply:
Ah, awesome, didn't know that. Thanks!